google_chat_ros/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(google_chat_ros)
+
+find_package(
+  catkin REQUIRED COMPONENTS
+  actionlib_msgs
+)
+
+catkin_python_setup()
+
+add_action_files(
+  FILES
+  GoogleChatREST.action
+)
+
+generate_messages(
+  DEPENDENCIES
+  std_msgs
+  actionlib_msgs
+)
+
+catkin_package()
+
+include_directories()
+
+# install
+# euslisp
+file(GLOB EUSLISP_SCRIPTS scripts/*.l)
+install(FILES ${EUSLISP_SCRIPTS}
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+# python
+file(GLOB PYTHON_SCRIPTS scripts/*.py)
+catkin_install_python(
+  PROGRAMS ${PYTHON_SCRIPTS}
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+# launch
+install(DIRECTORY launch/
```

Comment:
I would recommend to remove `/` from `launch/` and `/launch` from `${CATKIN_PACKAGE_SHARE_DESTINATION}/launch`, so that we can add more directories in future.
https://github.com/ros/urdf_tutorial/blob/26eef2c61def249d6164ca8159bfd1ff91ea7d50/CMakeLists.txt#L12-L14

Reply:
I'll fix